app/components/header/header.component.html

```diff
@@ -363,8 +369,8 @@ <h4 class="modal-title">Info</h4>
         <input type="text" id="origin-path" style=" width: 554px !important" placeholder="https://github.com/ORIGINAL_OWNER/ORIGINAL_OWNER_REPOSITORY.git">
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-primary" data-dismiss="modal" onClick="fetchFromOrigin()">Confirm</button>
-        <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+        <button type="button" class="btn btn-primary" data-dismiss="modal" onClick="fetchFromOrigin()">test</button>
```

Comment:
What's test for?

Reply:
I've added functionality to clear the input